Sources/TuistKit/Models/Target.swift

```diff
@@ -18,6 +18,8 @@ class Target: Equatable {
     // An info.plist file is needed for (dynamic) frameworks, applications and executables
     // however is not needed for other products such as static libraries.
     let infoPlist: AbsolutePath?
+    let mainStoryboard: String?
+    let launchScreenStoryboard: String?
```

Comment:
I used `String` because I don't need to know the location of the storyboard. Only it's name. I thought using `AbsolutePath` might confuse what I am asking for.